mlflow/sentence_transformers/__init__.py

```diff
@@ -221,6 +221,29 @@ def log_model(
     Log a ``sentence_transformers`` model as an MLflow artifact for the current run.
 
     :param model: A trained ``sentence-transformers`` model.
+
+        An example of using log_model for a sentence-transformers model and architecture:
+
+        from sentence_transformers import SentenceTransformer
+        import mlflow.sentence_transformers
+
+        model = SentenceTransformer("all-MiniLM-L6-v2")
+
+        signature = mlflow.models.infer_signature(
+            model_input=example_sentences,
```

Comment:
Can you define `example_sentences`?

Reply:
should this be done as a commented line in the code block or prior/after the code block?

Reply:
It can be pretty simple as a declared variable above the declaration of the `signature` variable within the same code block :)